src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/service/SubmissionServiceImpl.java

```diff
@@ -27,6 +27,8 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
+
+
```

Comment:
I thought it was 2 lines between import and class definition. weird your linter didn't catch this, or I'm dumb.